src/ws/test-remotectlcertificate.c

```diff
@@ -243,7 +254,7 @@ test_refresh_expired (TestCase *test,
   g_assert_cmpint (g_tls_certificate_verify (certificate, NULL, certificate), ==, G_TLS_CERTIFICATE_EXPIRED);
 
   /* call with --ensure again, refreshes the cert */
-  ret = cockpit_remotectl_certificate (4, argv);
+  ret = cockpit_remotectl_certificate (test->group ? 6 : 4, argv);
```

Comment:
...but this is a total no-go.  Can you replace that with `g_strv_length()`?

Reply:
The condition got dropped, but g_strv_length() is nice anyway. Thanks! Done.